src/main/java/htsjdk/samtools/cram/encoding/core/experimental/GammaIntegerCodec.java

```diff
@@ -15,58 +15,64 @@
  * limitations under the License.
  * ****************************************************************************
  */
-package htsjdk.samtools.cram.encoding;
+package htsjdk.samtools.cram.encoding.core.experimental;
```

Comment:
Please do not review the `experimental` Encodings/Codecs.  They are explicitly not up to standards.